### ISSUE TITLE
Idea: resource pack on client side

### ISSUE THREAD

**Issue by user-naa** [DorkixAzIgazi/armored-elytra#4]
It will be good if on client you need install only resource pack instead of mod.
P. S. Sorry for my English

**Comment by DorkixAzIgazi** [comment]
Yeah that would be nice but some [code](https://github.com/DorkixAzIgazi/armored-elytra/blob/main/src%2Fclient%2Fjava%2Fdorkix%2Farmored%2Felytra%2FArmoredElytraClient.java) is needed to get the armor data from the item. I don't think its possible to do this with only resourcepack json files.

**Comment by Wolfkid200444** [comment]
maybe support for Polymer?

**Comment by DorkixAzIgazi** [comment]
The latest version for 1.21.4 supports installing the icons via a resourcepack.
